.buildkite/commands/run-unit-tests.sh

```diff
@@ -4,10 +4,13 @@ echo "--- 🧪 Testing"
 set +e
 if [ "$1" == "wordpress" ]; then
     test_suite="testWordpressVanillaRelease"
+    test_log_dir="WordPress/build/test-results/*/*.xml"
 elif [ "$1" == "processors" ]; then
     test_suite=":libs:processors:test"
+    test_log_dir="libs/processors/build/test-results/test/*.xml"
 elif [ "$1" == "image-editor" ]; then
     test_suite=":libs:image-editor:test"
+    test_log_dir="libs/image-editor/build/test-results/testReleaseUnitTest/*.xml"
```

Comment:
I have to note that `image-editor` Unit Tests are executed for both release and debug, as can be seen from artefacts:

![Screenshot 2023-07-19 at 15 49 00](https://github.com/wordpress-mobile/WordPress-Android/assets/73365754/83afc216-e7ea-48bc-a26e-aecb976a8458)

However, only the release folder is copied, currently. The reason behind this is that the tests names and log files names are identical between those two folders, so moving them both to the same test suite in Test Analytics results in doubled entries with no way to track down which is which:

<img width="1179" alt="Screenshot 2023-07-19 at 20 44 55" src="https://github.com/wordpress-mobile/WordPress-Android/assets/73365754/e3169f67-4836-4fd1-989c-f0ebac9ce84f">

One of the solutions is creating two separate test suites for tests analytics, one being `Image Editor Release` and the other `Image Editor Debug`. I'm not sure, however, if this is really valuable 🤔 What are your thoughts?

Reply:
Yeah, I agree that it's probably not worth it to have both.

I'm actually even pondering if it makes sense to continue running the `image-editor` Unit Tests in `Debug` in the first place (as opposed to only run them in `Release`, instead of both configs)…

Reply:
I've asked about the necessity of running the debug ones in slack: p1689846526946439-slack-C04TBSWP09M